### PR TITLE
Added function filter_nth_entry()

### LINKED ISSUE #15: Add filtering by appointment
Add method that can filter a dataframe to only include the nth appointment of a study subject

### PR DESCRIPTION
**Pull request by p-mq**
Added function to filter by the nth entry of subjects. Defaults to keep the first entry of each subject, but can be specified to filter for the nth appointment and/or in reverse order.

Included tests and documentation

Closes #15

### CODE REVIEW

File: R/filter_nth_entry.R [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+#' @param entry_column character column identifying order of entries.
+#' That column can by of types Date, numeric, or any other type suitable for order()
+#' @param n integer number of entry to keep after ordering
+#' @param reverse_order logical when TRUE sorts entries last to first before filtering
+#'
+#' @return data.frame with <= 1 entry per subject
+#'
+#' @examples
+#' data <- data.frame(list(ID = rep(1:5, 3), encounter = rep(1:3, each=5), value = rep(4:6, each=5)))
+#'
```
Comment (Besttd) [inline R/filter_nth_entry.R]: Seems like the example is missing. Please add.

**Review by Besttd — CHANGES_REQUESTED** [review]
Please see my comment in R/filter_nth_entry.R

File: R/filter_nth_entry.R (hunk 1 above)
Comment (p-mq) [reply R/filter_nth_entry.R]: Updated code and mnual to include examples

**Review by Besttd — APPROVED** [review]
Thanks for fixing!

**Pull request merged**